WBooks/Library/LibraryViewModel.swift

```diff
@@ -49,4 +53,21 @@ class LibraryViewModel {
         return BookViewModel(book: book)
     }
     
+    func filterBooks(searchText: String) {
+        searchActive = searchText.isNotEmpty
+        
+        mutableFilteredBooks.value = books.value.filter({ (book: Book) -> Bool in
```

Comment:
I wrote it like this because it will be easier to change it to an repository call, if there is one in the future.